build.rs

```diff
@@ -0,0 +1,32 @@
+use std::process::Command;
+
+/// Pass git-describe through CARGO_GIT_VERSION env variable
```

Comment:
Why is it needed?

Reply:
I've found it useful in the past to embed the git commit in the binary so that when I'm looking through logs I can tell what version was running.
I can remove it if you don't think it's useful.